Apps/Medication/src/Properties/launchSettings.json

```diff
@@ -1,23 +1,23 @@
 {
```

Comment:
We use 4 spaces for indents for JSON files and most everything else, per our .editorconfig settings.  Not sure how this file got flipped back to 2 spaces.